lib/card.rb

```diff
@@ -38,4 +38,25 @@ class Card < ActiveRecord::Base
   TRACKED_FIELDS = %w(name type_id db_content trash)
   Loader.load_mods if count > 0
 
+
+  def self.find_each(options = {})
+    if block_given?
+      find_in_batches(options) do |records|
+        records.each { |record| yield record }
+      end
+    else
+      super(options)
+    end
+  end
```

Comment:
It doesn't change anything but if I don't overwrite find_each then it calls the original find_in_batches instead of Card.find_in_batches